jerry-debugger/jerry-client-ws.html

```diff
@@ -828,7 +828,23 @@
   {
     breakpoint = activeBreakpoints[index];
 
-    if (!breakpoint)
+    if (index == "all")
+    {
+      var found = false;
+
+      for (var i in activeBreakpoints)
+      {
+        delete activeBreakpoints[i];
+        found = true;
+      }
+
+      if (!found)
+      {
+        appendLog("No active breakpoints.")
+      }
+    }
+
```

Comment:
I think this form would be better, whats your opinion?
activeBreakpoints.length != 0 ? activeBreakpoints.splice(0) : appendLog("No active breakpoints.")